### ISSUE TITLE
Are tokens really single use?

### ISSUE THREAD

**Issue by martinthomson** [ietf-wg-privacypass/base-drafts#437]
> Unlike many authentication schemes in which a client will present the same credentials across multiple requests, tokens used with the "PrivateToken" scheme are single-use credentials, and are not reused. Spending the same token value more than once allows the origin to link multiple transactions to the same client. In deployment scenarios where origins send token challenges to request tokens, origins ought to expect at most one request containing a token from the client in reaction to a particular challenge.

I can think of many cases where a client wants to access a series of resources and use this scheme to access each.  I can think of two broad categories:

1. Error recovery.  If the client is unable to retrieve a resource for reasons unrelated to the token (a broken connection, say), they might want to retry the request.
2. Multiple resources within a single transaction or context.  Though it might be possible to only authorize a single request (the root HTML of a web page), HTTP servers often provide access control on a per-resource basis.  Having the token reused in that setting isn't harmful to privacy, particularly if there are other means of correlating requests (like the shared connection that is used to make all those requests).

In other words, I think that a non-reuse requirement could be problematic in the first case and unnecessary in the second.  Of course, loosening this requirement might make replay protections and whatnot more interesting, but that's something that can be managed.
